prom2json.go

```diff
@@ -140,6 +141,13 @@ func FetchMetricFamilies(
 	} else {
 		transport = &http.Transport{}
 	}
+	if skipServerCertCheck {
+		if transport.TLSClientConfig != nil {
+			transport.TLSClientConfig.InsecureSkipVerify = true
+		} else {
+			transport.TLSClientConfig = &tls.Config{InsecureSkipVerify: true}
+		}
+	}
```

Comment:
I think you can avoid this whole section by adding a line `InsecureSkipVerify: skipServerCertCheck,` after line 137, and by changing line 142 to
```go
transport = &http.Transport{
    TLSClientConfig: &tls.Config{InsecureSkipVerify: skipServerCertCheck},
}
```

Does that work?

Reply:
this would erase the configuration object which is created on line 139